src/main/app/Entity/Address.php

```diff
@@ -11,42 +11,42 @@ trait Address
      *
      * @var string
      */
-    private $addressStreet1;
+    protected $addressStreet1;
```

Comment:
Why not keeping it private?

Reply:
Because of : https://github.com/claroline/Claroline/blob/13.x/src/main/app/API/Finder/AbstractFinder.php#L166. There are other examples.

`property_exists` is always `false` when the prop is private. Same result with `ReflectionClass::hasProperty`.

Reply:
Thanks ^^